R/add-z.R

```diff
@@ -15,7 +15,7 @@ ps_sfc_add_z <- function(x, sfc_column = ps_active_sfc_name(x), z_column = "Elev
   x %<>% ps_activate_sfc(sfc_column)
   x %<>% cbind(sf::st_coordinates(x))
 
-  sfg <- do.call(list, purrr::map(1:nrow(x), function(y){
+  sfg <- do.call(list, purrr::map(seq_len(nrow(x)), function(y){
```

Comment:
Nice on the use of seq_len

Reply:
Although technically it should have been a separate PR to make review easier.

Reply:
noted, thank you